### DISCUSSION TITLE
Unable to open my files since choose Code with Bricks just to see

### DISCUSSION THREAD

**Discussion by didiersm** [pybricks/support#1310] [General]
All the issue is on the title 
All my opened files ask me for a valid member user!

**Comment by laurensvalk** [comment]
Pythons programs should remain accessible at any time. I'll look into it.

If this is a recurring issue, please download your scripts and edit them using the stable version: at https://code.pybricks.com/

**Reply by laurensvalk** [reply]
Found the issue and will push a fix later this afternoon.

**Reply by didiersm** [reply]
Cool you're the best ;)
Thk

**Reply by laurensvalk** [reply]
This was my mistake. A fix has been released but programs opened previously may still be affected. My apologies, and thanks for alerting me to this :slightly_smiling_face:

For the best and most stable experience, please use https://code.pybricks.com/ and use the beta only if you're curious to try out the latest (experimental) features. Thanks!

**Reply by didiersm** [reply]
You're welcome
Yes they are affected... never mind I was in test
Thanks for all you do
